docs/internal-documentation/distrib-testing/Ubuntu-nodjs-build-environment.md

```diff
@@ -0,0 +1,53 @@
+# Ubuntu nodjs Build Environment
+
+The requirements to build the Zonemaster-GUI distribution zip file are nodejs
+and npm. Below are instructions to create such an build environment on Ubuntu.
+
+Nodjs and npn are available from the [Node.js] official website. Minimal version
+of Nodejs is 10.0 but install the latest LTS version available. The process has
+been tested on Ubuntu 18.04, which we use here.
+
+1. Make a clean installation of Ubuntu 18.04.
+
+> Are we root at this stage?
+
+2. Update the package database.
+
+   ```sh
+   apt-get update
+   ```
+
+3. Create a new user.
+
+   ```sh
+   adduser username
+   ```
+
+4. Add user to the sudoer list.
+
+   ```sh
+   visudo
+   ```
+
+   Copy the contents of the ‘root’ in 'visudo' and change ‘root’ to ‘username’.
+
+
+5. Exit from the root shell.
+
+   ```sh
+   exit
+   ```
```

Comment:
If sections 3-4 are removed, this one is not necessary as you can use `sudo` in 2.